### ISSUE TITLE
Question: hydrate() returns an extra div on the server

### ISSUE THREAD

**Issue by rstacruz** [hashicorp/next-mdx-remote#123]
I noticed an issue where calling `hydrate()` on a server returns markup that looked like this:

```html
<div>
  <h2>...</h2>
  <p>...</p>
  <p>...</p>
</div>
```

But the extra div is gone when calling hydrate() on the browser:

```html
  <h2>...</h2>
  <p>...</p>
  <p>...</p>
```

Doing console.log shows that the `type` is 'div' on the server:

```js
export default function ArticlePage(props) {
  const { meta, mdxSource, id } = props
  if (!mdxSource) return null

  const content = hydrate(mdxSource, { components })
  console.log(content)

// On the server:
// { '$$typeof': Symbol(react.element},
//   type: 'div',
//   key: null
```

But in the browser, it seems to be rendering twice, with the second render showing a different type:

![image](https://user-images.githubusercontent.com/74385/116632702-232b5700-a99b-11eb-8783-2d5bbcd915e2.png)

I can provide more context if needed, but I'd love some pointers on where to start debugging this!

**Comment by jescalan** [comment]
Hey there! First of all just want to say that we use a good chunk of your open source work, so thank you for all the lovely things you have made. Second, would you mind giving [our upcoming major release](https://github.com/hashicorp/next-mdx-remote/pull/121) a try? I think it might resolve this issue for you 😀

**Comment by rstacruz** [comment]
Thanks @jescalan, it worked well! The new version consistently has a `<div>` and the second hydration is gone.

**Comment by jescalan** [comment]
That's great! We are planning on having that one released soon as stable, and when it is I will ping and close this issue out.

**Comment by BRKalow** [comment]
As @jescalan mentioned, this should now be fixed on latest! Please checkout the [3.0.0 release](https://github.com/hashicorp/next-mdx-remote/releases/tag/3.0.0) and let us know if you're still seeing the issue.
